aten/src/ATen/SparseTensorImpl.h

```diff
@@ -163,7 +143,6 @@ struct AT_API SparseTensorImpl : public TensorImpl {
     values_size.insert(values_size.end(), dense_size.begin(), dense_size.end());
     auto empty_values = values().type().tensor(values_size);
     set_indices_and_values_unsafe(empty_indices, empty_values);
-    refresh_numel();
```

Comment:
Err... are you sure?

Reply:
`set_sizes_and_strides(size.vec(), {})` will call `refresh_numel()` which should produce the correct `numel`, and we don't change the size of the sparse tensor after the `set_sizes_and_strides(size.vec(), {})` call.